src/main/java/org/example/Multi.java

```diff
@@ -0,0 +1,104 @@
+package org.example;
+
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.function.Supplier;
+import java.util.stream.IntStream;
+
+public class Multi {
+
+    private ExecutorService executorService = Executors.newFixedThreadPool(3);
```

Comment:
Move numbers/strings to constant with a good name. Otherwise, it's antipattern called magic number, https://refactoring.guru/replace-magic-number-with-symbolic-constant